ResearchKit/Common/ORKHeadlineLabel.m

```diff
@@ -53,6 +55,17 @@ + (UIFont *)defaultFont {
     return [self defaultFontInSurveyMode:NO];
 }
 
+- (instancetype)initWithFrame:(CGRect)frame {
+    [NSNotificationCenter.defaultCenter addObserverForName:CEVORKStepViewControllerViewWillAppearNotification object:nil queue:NSOperationQueue.mainQueue usingBlock:^(NSNotification * _Nonnull note) {
```

Comment:
Looks odd to me to set up the observer before calling super init. The more common pattern used elsewhere in ResearchKit (e.g. ORKAnswerTextField.m) would be:

```objc
self = [super initWithFrame:frame];
if (self) { /* add observer here */ }
return self;
```